src/components/navBar.tsx

```diff
@@ -14,7 +17,31 @@ const NavbarContainer = styled('div')`
     justify-content: space-between;
     align-items: center;
     padding: 0px 20px;
-`
+`;
+
+const TextFieldContainer = styled.div<{border: string}>`
+    height: 100%;
+    width: 100%;
+    border: 1px solid ${props  => props.border};
```

Comment:
if you are passing a hard-coded border color, y giving it down the props ?? and even if you are doing so, make it dynamic, when the div is focused change the color using these props